### DISCUSSION TITLE
Emissary Executor command error

### DISCUSSION THREAD

**Discussion by kotalakshman** [argoproj/argo-workflows#9520] [Q&A] [answered]
This step is in Error state with this message: `task 'covertype-classifier-training-n9gws.bigquery-query' errored: when using the emissary executor you must either explicitly specify the command, or list the image's command in the index: https://argoproj.github.io/argo-workflows/workflow-executors/#emissary-emissary`

**Comment by terrytangyuan** [comment, marked as answer]
Take a look at the error message: "you must either explicitly specify the command, or list the image's command in the index: https://argoproj.github.io/argo-workflows/workflow-executors/#emissary-emissary"
